src/components/courseSettings/ManageStaffPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  ListGroup,
+  ListGroupItem,
+} from 'reactstrap'
+import FlipMove from 'react-flip-move'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faUsers } from '@fortawesome/free-solid-svg-icons'
+import AddStaff from '../AddStaff'
+
+import RemoveableUserItem from '../RemoveableUserItem'
+
+interface ManageStaffPanelProps {
+  course: {
+    id: number
+    name: string
+    staff: number[]
+  }
+  users: {
+    id: number
+    netid: string
+    name: string
+  }[]
+  removeCourseStaff: (courseId: number, userId: number) => void
+  addCourseStaff: (courseId: number, netid: string, name: string) => void
+}
+
+const ManageStaffPanel = ({
+  course,
+  users,
+  removeCourseStaff,
+  addCourseStaff,
+}: ManageStaffPanelProps) => {
+  let newUsers
+  if (course.staff && course.staff.length > 0) {
+    newUsers = course.staff.map(id => {
+      const user = users[id]
+      return (
+        <RemoveableUserItem
+          key={user.id}
+          onRemove={(userId: number) => removeCourseStaff(course.id, userId)}
+          {...user}
+        />
+      )
+    })
+  } else {
+    newUsers = (
+      <ListGroupItem className="text-center text-muted pt-4 pb-4">
+        This course doesn&apos;t have any staff yet
+      </ListGroupItem>
+    )
+  }
+
+  return (
+    <Card className="mb-3">
+      <CardHeader>
+        <CardTitle tag="h5" className="mb-0">
+          <FontAwesomeIcon icon={faUsers} className="mr-2" /> Manage Staff
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <AddStaff
+          onAddStaff={(staff: Record<string, string>) => {
+            const { netid, name } = staff
+            addCourseStaff(course.id, netid, name)
+          }}
+        />
+
+        <ListGroup flush className="position-relative">
```

Comment:
With the `flush` prop, this should be sitting as a direct child of `Card` and not nested inside `CardBody`.

Reply:
Whenever I make `ListGroup` a direct child of `Card`, this is what happens:
![image](https://user-images.githubusercontent.com/13933251/68962531-6e977880-079a-11ea-8ebc-31fd55595fbd.png)

Reply:
Hmmm, interesting. Gotcha, we can leave this as is for now. Can you file another issue to capture this so we can look into this and other styling improvements in the future?

Reply:
Will do!